yii2 advanced Lesson 1

Gromoboy opened the pull request:
| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes/no
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

Gromoboy committed:
Добавлены пользователи в бд (таблица user) через миграцию

Gromoboy committed:
установлен модуль adminlte, применен к бэкэнду

TalismanFR commented:
Добрый день, Александр!
Репозиторий вижу. 

AdminLte привязан путем переопределения пути до view. Это может создать проблемы в будущем, т.к. вы не можете создавать файлы в vendor. Лучше просто скопировать файлы из @vendor/dmstr/yii2-adminlte-asset/example-views/yiisoft/yii2-app в ваш локальный views.

Gromoboy committed:
Отменено переопределение view

на пример темы adminlte в папке vendor.
Через переминование файлов подготовлена установка adminlte копированием файлов из примера(для сохранения старых версий)

Gromoboy committed:
Переустановлен adminlte

уже путём копирования файлов из примера, а не через смены пути view
